Release_Process.md

```diff
@@ -0,0 +1,42 @@
+# Confidential Container Release Process
+
+## Process
+- Code Freeze
+- Issue created from [Release Check list template](https://github.com/confidential-containers/community/issues/new?assignees=&labels=&template=release-check-list.md&title=%5BRelease%5D+Check+list+for+v%3CTARGET_RELEASE%3E)
+- "[Release Owner](https://github.com/confidential-containers/community/wiki/Release-Team-Rota)" identifies owners for Check List Items
+- Check List Items completed
+- Release Complete
+- Removal of Code Freeze and release announced.
+
+Release Dates will be set mid week and no one should feel an expectation to work unsocial hours to meet a release date. Communication and expectation setting is important and can be used to involve more people from the community to address any Issues blocking release. From the point of code freeze the release is being built so the release date is an estimate. Releasing early or late along with ideas for improvements/changes for future releases will be reflected on after release.
+
+
+## Requirements to help with Release Process
+- Availability during the release period to complete Tasks 
+- **TODO** What access is needed/suggested?
+- GitHub permissions to push tags and create releases in CoCo repositories
+
+## Expectations on Release Owners
+The release owner is not expected to complete the Release Check List alone and the majority of tasks on the check list can be delegated. Delegation will help others become confident assisting with the release and share the burden but does rely on community members volountering.
+
+The key responsibility which can not be delegated is communication and oversight
```

Comment:
Would it make sense to create a general Slack channel to help coordinate and oversee the release process?

Reply:
I like that idea, Do we feel people would add themselves to the channel? and it would be used. not sure what others involved in the release process previously feel.
I think I'd like to try it and we can document the channel name.
We can always link across to a thread in it from general confidential containers channel if problems need fixed by the wider community , rather than threads on progressing through the steps etc.

Reply:
In the past we have used threads in the CNCF and Kata workspaces.

Reply:
Added reference to threads